packages/sophos/data_stream/utm/elasticsearch/ingest_pipeline/default.yml

```
@@ -1,87 +1,168 @@
-      properties:
-          - asn
-          - organization_name
-      ignore_missing: true
-  - rename:
-      field: source.as.asn
-      target_field: source.as.number
-      ignore_missing: true
-  - rename:
-      field: source.as.organization_name
-      target_field: source.as.organization.name
-      ignore_missing: true
+      tag: set_ecs_version
   - rename:
-      field: destination.as.asn
-      target_field: destination.as.number
+      field: message
+      target_field: event.original
       ignore_missing: true
+      tag: rename_event_original
+  - set:
+      field: event.category
+      value: [network]
+      tag: set_event_category
+  - set:
+      field: event.type
+      value: [info]
+      tag: set_event_type
+  - set:
+      field: event.kind
+      value: event
+      tag: set_event_kind
+
+  # Parse the event header: populates the timestamp, hostname, and process info
+  # from the header and stores the message in _tmp.raw_data
+  - grok:
+      field: event.original
+      if: ctx.event?.original != null
+      pattern_definitions:
+        SOPHOS_TIMESTAMP: (?:%{YEAR}:%{MONTHNUM}:%{MONTHDAY}-%{HOUR}:%{MINUTE}:%{SECOND})
+      patterns:
+        - "^%{SOPHOS_TIMESTAMP:_tmp.timestamp} (?:%{HOSTNAME:host.hostname}) %{DATA:process.name}(?:\\[%{POSINT:process.pid:long}\\])?: %{GREEDYDATA:_tmp.raw_data}"
+      tag: grok_syslog_header
+      on_failure:
+        - append:
+            field: error.message
+            value: "fail-{{{ _ingest.on_failure_processor_tag }}}"
+        - fail:
+            message: "Processor {{ _ingest.on_failure_processor_type }} with tag {{ _ingest.on_failure_processor_tag }} in pipeline {{ _ingest.on_failure_pipeline }} failed with message: {{ _ingest.on_failure_message }}"
+  # Process timestamp and timezone if configured (default UTC)
   - rename:
-      field: destination.as.organization_name
-      target_field: destination.as.organization.name
+      if: ctx._conf?.tz_offset != null
+      field: _conf.tz_offset
+      target_field: event.timezone
       ignore_missing: true
+      tag: rename_conf_timezone
+  - set:
+      field: event.timezone
+      value: UTC
+      override: false
+      tag: set_default_timezone
+  - date:
+      if: ctx?._tmp?.timestamp != null
+      timezone: "{{{ event.timezone }}}"
+      field: _tmp.timestamp
+      formats:
+        - yyyy:MM:dd-HH:mm:ss
+      tag: date_set_timestamp
+      on_failure:
+        - append:
+            field: error.message
+            value: "fail-{{{ _ingest.on_failure_processor_tag }}}"
+        - fail:
+            message: "Processor {{ _ingest.on_failure_processor_type }} with tag {{ _ingest.on_failure_processor_tag }} in pipeline {{ _ingest.on_failure_pipeline }} failed with message: {{ _ingest.on_failure_message }}"
+
+  # Choose the pipeline based on the log source
+  - set:
+      field: event.provider
+      value: 'dns'
+      if: ctx?.process?.name != null && ctx?.process?.name == 'named'
```

```suggestion
      if: ctx.process?.name != null && ctx.process.name == 'named'
```
`ctx.process.name` is non-null by the time we get to the second expression, similar below.